examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/SchemaTest.kt

```diff
@@ -56,7 +56,7 @@ class User: RealmObject {
     @PrimaryKey
     var _id: ObjectId = ObjectId()
     var name: String = ""
-    var posts: RealmList<Post>? = null
+    var posts: RealmList<Post>? = realmListOf()
```

Comment:
If Realm lists cannot be null, should we remove the `?` here?

```suggestion
    var posts: RealmList<Post> = realmListOf()
```